Automattic/simplenote-ios
# Nukes IS_XCODE_11 guards

### Details:
In this PR we're removing every IS_XCODE_11 guard we've got, both, in ObjC and Swift.

### Testing:
- Verify the project builds correctly
- Verify the code makes sense!
- Run a smoke test over the app
